guardian doc for admins

angelaLazar opened the pull request:
### Your checklist for this pull request

Please review the [guidelines for contributing](https://github.com/auth0/docs#contributing) to this repository.
- [ ] Content conforms to our [Contributing Guidelines](https://github.com/auth0/docs#contributing-guidelines)
- [ ] If applicable, you have added details to the [update feed](https://github.com/auth0/docs/tree/master/updates) 
### Description

Please describe your pull request.

Thank you!

angelaLazar committed:
guardian doc for admins

angelaLazar committed:
guardian doc for admins

dafortune commented on articles/multifactor-authentication/guardian/admin-guide.md:
```diff
@@ -0,0 +1,70 @@
+
+Users that were previously registered before you enable MFA, will complete the same process as new users on their next login.
+
+By enabling **Push Notifications** for MFA, you will also be enabling the option to use Google Authenticator instead of Guardian. To use this option, your users must have the Google Authenticator app downloaded to their mobile device, the app is available for both Android and iOS. When a new user signs up, or a user is logging in for the first time since enabling MFA there will be a code to be sc ...
```
> They need to have this recovery code to login without their mobile device. If they lose both the recovery code and their mobile device, you will need to reset their MFA [see below](/admin-guide#reset-mfa-for-a-user) on how to do this.

I know the process is the same, and someone might have a different opinion, but it sounded kind of repetitive when I read it first time.

dafortune commented:
Just a comment. Looks good to me

dafortune commented:
Would make sense to include some screenshots from Guardian dashboard?

woloski commented:
I dont see any mention about SMS

dafortune commented:
What about Guardian audit logs? I think we should mention them

hzalaz commented on articles/multifactor-authentication/guardian/admin-guide.md:
```diff
@@ -0,0 +1,70 @@
+ 
+     if (context.connection !== 'foo.com'){
+         context.multifactor = {
+             provider: 'guardian', //required
```
We should always suggest to add `ignoreCookies: true` in the multi factor options

woloski replied:
And also explain what it means to set this

mpaktiti committed:
New doc for logentries extension (#1368)

Commit:
Appliance Monitoring (#1371)

* Initial commit

* Add monitoring options

* Add section on testall endpoint

* Add hyperlink

* Add section on authenticated endpoints

* Update testall info

* Update testall info

* Add details on synthetic transactions

* Fix landing page

Catografix committed:
[update] Mostly new screenshots and instructions (#1367)

Catografix committed:
[edit] fixed api call, added description (#1365)

angelaLazar committed:
Guardian User Guide (#1326)

* Guardian User Guide

* fixing up Guardian user doc

* addressing comments in guardian user doc

angelaLazar committed:
updating screenshot (#1375)

angelaLazar committed:
adding audit event info plus addressing comments for Admin docs

angelaLazar committed:
adding audit event info plus addressing comments for Admin docs

angelaLazar commented:
I added a section on SMS, audit events, updateCookies and added more screenshots

ntotten committed:
Update user-profile-details.md

mpaktiti committed:
Add warning for custom databases (#1383)

Add warning that in case of custom DB the user should first check if the user exists and then invoke the API.

Also, removed some spaces in an attempt to fix the request's postData that are currently not displayed.

mpaktiti committed:
Update steps and screenshots (#1384)

woloski commented on articles/multifactor-authentication/guardian/admin-guide.md:
```diff
@@ -0,0 +1,123 @@
+
+Users that were previously registered before you enable MFA, will complete the same process as new users on their next login.
+
+By enabling **Push Notifications** for MFA, you will also be enabling the option to use Google Authenticator instead of Guardian. To use this option, your users must have the Google Authenticator app downloaded to their mobile device, the app is available for both Android and iOS. When a new user signs up, or a user is logging in for the first time since enabling MFA there will be a code to be sc ...
```
This paragraph is too complex and should be somwhere else, not on Guardian intro. Like a note saying that Guardian behind the scenes uses the same mechanism that Google Authenticator (or any TOTP Authenticator) uses, so for phones where there is no Guardian (e.g. Windows Phones), you can scan the QR code using Microsoft Authenticator. Actually, I would make a special section explaining the support on different types of devices and versions of OS.

Commit:
Add 2 docs to onboarding (#1373)

* create cloud-sprint.md in onboarding

* create appliance-sprint.md in onboarding

Commit:
Fixed markdown

Commit:
small fix

Commit:
Another typo

Commit:
Fix details on Appliance dashboard docs (#1386)

hzalaz committed:
[edit] Add note for RN 0.26 flag issue (#1374)

hzalaz committed:
Update version of CocoaPods & React Native (#1372)

nicosabena committed:
Update warning on popup mode bug. (#1369)

Updated warning on popup mode with IE problem and link to issue on ask.auth0.com.

angelaLazar committed:
adding audit events to guardian dev doc (#1385)

angelaLazar committed:
Adding info on Guardian alternatives

angelaLazar committed:
Adding info on Guardian alternatives

woloski commented on articles/multifactor-authentication/guardian/admin-guide.md:
```diff
@@ -0,0 +1,130 @@
+
+In the [Logs](${uiURL}/#/logs) section of the dashboard you can see the various events related to your users signing up and signing in using MFA.
+
+![](/media/articles/mfa/logs.png)
```
I don't see Guardian Logs in this screen

angelaLazar replied:
I updated the screenshot to have more guardian events

angelaLazar committed:
replaced screenshot

angelaLazar committed:
replaced screenshot

angelaLazar committed:
updating admin docs

angelaLazar committed:
Merge remote-tracking branch 'origin/guardian-admin' into guardian-admin

# Conflicts:
#	articles/multifactor-authentication/guardian/admin-guide.md

angelaLazar commented:
closing in favor of: https://github.com/auth0/docs/pull/1398
